setup.py

```diff
@@ -9,6 +9,8 @@
 
 agents = glob.glob(os.path.join(pkg_dir, "*_agent.*"))
 assert len(agents) > 0, "Agents not compiled; run “npm install && npm run build” in agents/*/"
+os.system("python ./build_aux/build_treesitter.py")
+package_data = agents + ["treesitter.so"]
```

Comment:
I think we should tie into the build system so we don't do this when just running setup.py to query for something.

Reply:
I think this is the only way that a source distribution would build correctly?